crates/goose-cli/src/cli.rs

```diff
@@ -672,6 +672,14 @@ enum Command {
         /// Open browser automatically
         #[arg(long, help = "Open browser automatically when server starts")]
         open: bool,
+
+        /// Authentication token for both Basic Auth (password) and Bearer token
+        #[arg(
+            long,
+            help = "Optional authentication token to secure the web interface",
+            long_help = "Optional token for authentication. Can be used as password in Basic Auth (username ignored) or as Bearer token in API calls. If not provided, no authentication is required."
+        )]
+        auth_token: Option<String>,
```

Comment:
can you remove the comment and where it says it is optional?

Reply:
ok done